sensor/kubernetes/complianceoperator/updater.go

```diff
@@ -116,6 +150,11 @@ func (u *updaterImpl) collectInfoAndSendResponse() bool {
 	info := u.getComplianceOperatorInfo()
 	u.complianceOperatorNS = info.GetNamespace()
 
+	// Register compliance operator objects for diagnostic bundles if it was found.
+	if info.GetIsInstalled() && info.GetNamespace() != "" {
```

Comment:
I think it might be good to check also if the `info.StatusError` is nil. `getComplianceOperatorInfo` might return that is installed and a namespace but still have errors if the sensor doesn't have write access or if some CRDs are missing.

Reply:
I thought about that too. When hitting `StatusErrors` we might **want** a diagnostic bundle to investigate the issue.
I've imagined a situation where we  do have `StatusErrors`, get the diagnostic bundle, and then no resources were added to it.